DylibBinder/DBFunc.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using SwiftReflector;
+using SwiftReflector.Demangling;
+using SwiftReflector.Inventory;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DylibBinder {
+	internal class DBFunc : IAssociatedTypes {
+		public string Name { get; }
+		public bool IsStatic { get; }
+		public bool IsProperty { get; }
+		public string ReturnType { get; }
+		public bool HasThrows { get; }
+		public bool IsMutating { get; }
+		public bool HasInstance { get; }
+
+		public string PropertyType { get; } = string.Empty;
+		public bool IsPossiblyIncomplete { get; } = false;
+		public OperatorType OperatorKind { get; } = OperatorType.None;
+		public TypeAccessibility Accessibility { get; } = TypeAccessibility.Public;
+		public bool IsFinal { get; } = false;
+		public bool IsDeprecated { get; } = false;
+		public bool IsUnavailable { get; } = false;
+		public bool IsOptional { get; } = false;
+		public bool IsRequired { get; } = false;
+		public bool IsConvenienceInit { get; } = false;
+		public string ObjcSelector { get; } = string.Empty;
+
+		public DBParameterLists ParameterLists { get; }
+		public DBGenericParameters GenericParameters { get; }
+		public DBAssociatedTypes AssociatedTypes { get; } = new DBAssociatedTypes ();
+
+		public DBFunc (TLFunction tlf, bool isMutating = false, bool isProtocol = false)
+		{
+			Name = tlf.Name.Name;
+			IsStatic = GetStaticStatus (tlf.Signature);
+			IsProperty = false;
```

Comment:
Looking at this, IsProperty, is it ever true for this type?

Reply:
For this constructor, it is false, but for the constructor that takes a DBProperty, it is true